variants/coldwar/coldwar.go

```diff
@@ -16,7 +16,11 @@ const (
 var Nations = []godip.Nation{USSR, NATO}
 
 var ColdWarVariant = common.Variant{
-	Name:       "Cold War",
+	Name: "Cold War",
+	NationColors: map[godip.Nation]string{
```

Comment:
@JorenC This is how you define nation colors in variants. Leave empty or don't populate a particular nation and I'll make the client color it with the old default. NB: Make sure to change these example colors ;)